queries.md

```diff
@@ -571,6 +571,8 @@ The `whereTime` method may be used to compare a column's value against a specifi
                     ->whereTime('created_at', '=', '11:20:45')
                     ->get();
 
+> {note} These methods can result in significant performance degration, because the database can not make use of indexes for the `where` contions!
```

Comment:
```suggestion
> {note} These methods can result in significant performance degration, because the database can not make use of indexes for the `where` conditions!
```